sirepo/exporter.py

```diff
@@ -60,12 +60,21 @@ def _create_zip(sim, out_dir, qcall):
     data = simulation_db.open_json_file(sim.type, sid=sim.id, qcall=qcall)
     simulation_db.update_rsmanifest(data)
     data.pkdel("report")
+    if hasattr(sim.template, "add_fdir"):
+        sim.template.add_fdir(data, sim)
     files = sim_data.get_class(data).lib_files_for_export(data, qcall=qcall)
     for f in _python(data, sim, qcall):
         files.append(f)
     with sirepo.util.write_zip(str(path)) as z:
         for f in files:
-            z.write(str(f), f.basename)
+            if hasattr(sim.template, "export_filename"):
+                n = sim.template.export_filename(
+                    data.get("fDir", sim.filename.replace(".zip", "")),
+                    f.basename,
```

Comment:
The data.get("fDir") and sim.filename.replace parts are too specific and should be implementation left to template.srw. I'll update the branch with a change for this.